documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -3,43 +3,60 @@
 // assembly-configuring-kafka-listeners.adoc
 
 [id='proc-accessing-kafka-using-ingress-{context}']
-= Accessing Kafka using ingress
+= Accessing Kafka using ingresses
 
-This procedure shows how to access a Strimzi Kafka cluster from an external client outside of Kubernetes using Nginx Ingress.
+[role="_abstract"]
+Use an {NginxIngressController} to access a Strimzi Kafka cluster from clients outside the Kubernetes cluster. 
+The Nginx Ingress Controller supports connections from HTTP and HTTPS-based applications.
```

Comment:
This is confusing. You should either remove it or elaborate on the TLS passthrough and its challenges right here. Because otherwise people might look at this and instead of continue reading say hey, I can use my HTTP client here.

Reply:
Removed